src/main/java/org/broadinstitute/hellbender/tools/FixCallSetSampleOrdering.java

```diff
@@ -0,0 +1,149 @@
+package org.broadinstitute.hellbender.tools;
+
+import com.google.common.collect.Lists;
+import htsjdk.samtools.util.CloserUtil;
+import htsjdk.variant.variantcontext.VariantContext;
+import htsjdk.variant.variantcontext.writer.VariantContextWriter;
+import htsjdk.variant.vcf.VCFHeader;
+import htsjdk.variant.vcf.VCFHeaderLine;
+import org.apache.commons.collections.CollectionUtils;
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.barclay.argparser.BetaFeature;
+import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.cmdline.programgroups.VariantProgramGroup;
+import org.broadinstitute.hellbender.engine.*;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.tools.genomicsdb.GenomicsDBImport;
+import org.broadinstitute.hellbender.utils.io.IOUtils;
+
+
+import java.io.File;
+import java.nio.file.Path;
+import java.util.*;
+import java.util.stream.Collectors;
+
+@BetaFeature
+@CommandLineProgramProperties(summary = "fix the sample names in a vcf that ran through the GenomicsDBImport when " +
+        "the batch ordering bug was present, this will restore the correct sample names provided it is given the exact sample name mapping / vcf ordering" +
+        "and batch sized that was used in the initial import" +
+        "see https://github.com/broadinstitute/gatk/issues/3682 for more information",
```

Comment:
Add a space and period between "import" and "see"

Reply:
done